proguard/proguard-support-v7-appcompat.pro

```diff
@@ -0,0 +1,7 @@
+-keep public class android.support.v7.widget.** { *; }
```

Comment:
Just a note : 
v7-appcompat provide no consumer proguard files, I'm not sure how we are using this class though, so that this is okay. 
`whyareyoukeeping` might be able to improve these settings. 💭